### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by wy414012** [burst-apps-team/burstpool#41]
The build was passed, and the build failed because of an error caused by insufficient npm installation permissions. 
However, I don't know how to add it to the automatic build configuration file.

**Comment by harryjph** [comment]
Are you running as root? It will install npm if it is not already installed.

**Comment by wy414012** [comment]
Yes, it is run as root, but you need to manually install the skip permission.

**Comment by harryjph** [comment]
I'm not sure what you mean by the skip permission?
